common/djangoapps/third_party_auth/models.py

```diff
@@ -67,8 +67,13 @@ def clean_json(value, of_type):
 
 
 def clean_username(username=''):
```

Comment:
[question] The PR description made it seem like we had dropped a call to `clean-username` somewhere in the past. Is that correct? Can/should we restore that call if it has now been updated to handle all cases?

Reply:
Yes, I thought about this. But additional call seems to be useless in our case. I mean, `call_username` will be called anyway while initializing the registration form so I don't think we need to call it once more before (in the TPA pipeline)